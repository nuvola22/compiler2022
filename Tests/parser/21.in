var a: Real = 10;
    b: Real = 11;
    c: Real = 12;
    d: Real = 13;

begin
    writeln(2 <> 3);
    writeln(2 < 3);
    writeln(2 <= 3);
    writeln(2 >= 3);
    writeln(2 > 3);
    writeln(2 = 3);

end.