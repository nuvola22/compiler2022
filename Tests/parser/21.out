      program
         variable
            a
            Real
            10
         variable
            b
            Real
            11
         variable
            c
            Real
            12
         variable
            d
            Real
            13
         begin
            call
               writeln
               <>
                  2
                  3
            call
               writeln
               <
                  2
                  3
            call
               writeln
               <=
                  2
                  3
            call
               writeln
               >=
                  2
                  3
            call
               writeln
               >
                  2
                  3
            call
               writeln
               =
                  2
                  3
